spring-cloud-gateway-core/src/main/java/org/springframework/cloud/gateway/config/HttpClientProperties.java

```diff
@@ -51,7 +51,7 @@
 public class HttpClientProperties {
 
 	/** The connect timeout in millis, the default is 45s. */
-	private Integer connectTimeout;
+	private Integer connectTimeout = 45000;
```

Comment:
The default comes from HttpClient. Please remove this and the test and leave just the additional metadata.